core-services/release-controller/_releases/release-origin-4.7.json

```diff
@@ -16,7 +16,7 @@
     "vsphere":{
```

Comment:
We should rename this file release-okd-*

Reply:
https://github.com/openshift/release/pull/17752